homeassistant/components/becker/translations/en.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file can be automatically generated by this script:
```bash
python3 -m script.translations develop
```
It prevents errors such as a blank line at the end, tabs instead of spaces, etc...